styles/global.css

```diff
@@ -76,20 +120,73 @@ body {
 }
 
 .dropdown-content {
-  background-color: rgb(253, 252, 252);
+  background-color: black;
+  /* hide dropdown content by default */
   display: none;
   width: 100px;
+  /* make content appear below dropdown button */
   position: absolute;
+  top: 80%;
 }
 
 .dropdown-content a {
   /* start with new line */
   display: block;
-  color: black;
+  color: white;
+  padding: 10px;
 }
 .dropdown:hover .dropdown-content {
   /* without this, display: none for content */
   display: block;
   padding: 12px;
   text-align: left;
 }
+.search-container {
+  border-radius: 4px;
+  padding-left: 6px;
+  padding-right: 6px;
+}
+.search-box {
+  background-color: black;
+  color: white;
+  height: 100%;
+  font-family: "Hind", sans-serif;
```

Comment:
Instead of wrting the font family in many places. You can create a custom variable like so:
```
:root {
  --main-font: "Hind", sans-serif;
}

body {
  font-family: var(--main-font);
}

.search-box {
  background-color: black;
  color: white;
  height: 100%;
  font-family: var(--main-font);
  font-weight: 600;
  border: none;
}
```

Reply:
so for our case we can make `--heading-sans` and `--paragraph-sans` for Hind and Open Sans